sopt/src/main/java/dagr/sopt/cmdline/ClpAnnotation.java

```diff
@@ -1,7 +1,7 @@
 /*
  * The MIT License
  *
- * Copyright (c) 2015 Fulcrum Genomics LLC
+ * Copyright (c) $year Fulcrum Genomics
```

Comment:
I don't think we should have `$year` here, we should have the actual year this file was written.
Also, in some places you've added LLC and in some places you've taken it away.